src/styles/Homepage.styles.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import colors from "./colors";
+import fonts from "./font_names";
+
+export const HeroContainer = styled.div`
+  background-size: cover;
+  background-color: black;
+`;
+
+export const HeroContent = styled.div`
+  position: relative;
+  z-index: 1;
+
+  padding-top: 233px;
+  margin-left: 6vw;
+  img {
+    width: 50vw;
+  }
+`;
+
+export const HeroDescription = styled.div`
+  font-family: itc-avant-garde-gothic-pro, sans-serif;
+  font-size: 27px;
```

Comment:
add these + sizes to font constants and pull from there if they're going to be used elsewhere on the site!